modules/apps/commerce/commerce-product-api/src/main/java/com/liferay/commerce/product/service/CPAttachmentFileEntryLocalService.java

```diff
@@ -128,6 +128,9 @@ public CPAttachmentFileEntry createCPAttachmentFileEntry(
 	public PersistedModel createPersistedModel(Serializable primaryKeyObj)
 		throws PortalException;
 
+	public void deleteCPAttachmentFileEntries(long fileEntryId)
```

Comment:
why don't you use this method directly?

Reply:
Hi @andrea-ale-sbarra , I am not using the persistence directly because I believe we should still go through the logic in 		cpAttachmentFileEntryLocalService.deleteCPAttachmentFileEntry(
				cpAttachmentFileEntry);